src/portable/synopsys/dwc2/dcd_dwc2.c

```diff
@@ -348,6 +361,11 @@ static void edpt_schedule_packets(uint8_t rhport, const uint8_t epnum, const uin
 
   const bool is_dma = dma_device_enabled(dwc2);
   if(is_dma) {
+    if (dir == TUSB_DIR_IN && total_bytes != 0) {
+      // CACHE HINT
+      // The xfer->buffer has new data for Host, move it to memory for DMA to transfer it
+      dcd_dcache_clean(xfer->buffer, total_bytes);
```

Comment:
I can see that, the functions `cdc_cache_clean` and `dcd_dcache_invalidate` definitions are enclosed into `#if` in `dwc2_esp32.h`

Then, `dcd_dcache_invalidate` function call is placed in this file within the same `#if` as it's definition
But the `dcd_dcache_clean` function call is not. Should we need to add a `#if` for `dcd_dcache_clean` call?